AvalonStudio/AvalonStudio/Controls/QuickCommanderViewModel.cs

```diff
@@ -0,0 +1,230 @@
+                {
+                    SelectedIndex++;
+                }
+            });
+
+            EnterCommand = ReactiveCommand.Create();
+            EnterCommand.Subscribe(_ =>
+            {
+                IsVisible = false;
+            });
+
+            EscapeCommand = ReactiveCommand.Create();
+            EscapeCommand.Subscribe(_ =>
+            {
+                if (_shell.DocumentTabs.TemporaryDocument != null)
+                {
+                    if (_shell.DocumentTabs.TemporaryDocument.SourceFile == SelectedResult?.Model)
+                    {
+                        _shell.RemoveDocument(_shell.DocumentTabs.TemporaryDocument);
+                    }
+                }
+                
+                IsVisible = false;
+            });
+
+            _shell = IoC.Get<ShellViewModel>();
+        }        
+
+        private async Task ProcessQuery(string query)
+        {
+            if (_shell?.CurrentSolution == null)
+            {
+                return;
+            }
+
+            if (string.IsNullOrEmpty(query))
+            {
+                Dispatcher.UIThread.InvokeAsync(() =>
+                {
+                    ResultsVisible = false; 
+                    _results.Clear();
+                });
+            }
+            else
+            {
+                query = query.ToLower();
+
+                var list = new List<SearchResultViewModel>();
+
+                await Task.Run(() =>
+                {
+                    foreach (var project in _shell.CurrentSolution.Projects)
+                    {
+                        project.SourceFiles.Select(sf =>
+                        {
+                            //var match = FuzzyMatch.fuzzy_match(query, sf.Project.Location.MakeRelativePath(sf.Location), out int score, out string format);
+                            var match = FuzzyMatch.StringMatch(sf.Project.Location.MakeRelativePath(sf.Location), query, null);
+                            if(match != null)
+                            {
+                                return new Tuple<bool, int, string, ISourceFile, List<FuzzyMatch.Range>>(true, match.matchQuality, match.label, sf, match.stringRanges);
+                            }
+
+                            return new Tuple<bool, int, string, ISourceFile, List<FuzzyMatch.Range>>(false, 0, "", sf, null);
+                        }).Where(tp => tp.Item1).Select(tp =>
+                        {
+                            List<Avalonia.Media.FormattedTextStyleSpan> list_spans = new List<FormattedTextStyleSpan>();
+                            int index_start = 0;
+                            foreach(var range in tp.Item5)
+                            {
+                                if(range.matched)
+                                {
+                                    Avalonia.Media.FormattedTextStyleSpan span = new FormattedTextStyleSpan(index_start+tp.Item4.Project.Name.Length+1, range.text.Length+1, Brushes.Red);
```

Comment:
Add an IBrush field and in constructor create it by using Brushed.Parse("#AARRGGBB")